File: scripts/check-db.py
#!/usr/bin/env python3
import sqlite3

conn = sqlite3.connect('./state/database.db')
cursor = conn.cursor()

print('\n=== Modules Table ===')
cursor.execute('SELECT name, path, enabled FROM modules')
modules = cursor.fetchall()
for module in modules:
    print(f"Module: {module[0]}, Path: {module[1]}, Enabled: {module[2]}")

print('\n=== CLI Commands Table ===')
cursor.execute('SELECT module_name, command_name, command_path FROM cli_commands LIMIT 10')
commands = cursor.fetchall()
for cmd in commands:
    print(f"Module: {cmd[0]}, Command: {cmd[1]}, Path: {cmd[2]}")

conn.close()

File: scripts/check-db.js
const Database = require('better-sqlite3');
const db = new Database('./state/database.db', { readonly: true });

console.log('\n=== Modules Table ===');
const modules = db.prepare('SELECT name, path, enabled FROM modules').all();
console.log(modules);

console.log('\n=== CLI Commands Table ===');
const commands = db.prepare('SELECT * FROM cli_commands LIMIT 10').all();
console.log(commands);

db.close();